code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -157,6 +157,20 @@
 	..()
 	M.intoxication += removed*0.02 //Very slightly alcoholic
 
+/datum/reagent/nutriment/coating/vegemite
+	name = "Vegemite"
+	cooked_name = "burnt Vegemite"
+	id = "vegemite"
+	color = "#0c0901"
```

Comment:
Are you sure that colour is correct ? - Its almost black

Reply:
I googled it - and I think it looks accurate.

Reply:
Vegemite is black in the jar at high concentrations. When spread out on bread, and so fourth, it's brown.